server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/sql/Db2SqlExecLexer.g4

```diff
@@ -535,6 +535,7 @@ PARAMETER : P A R A M E T E R;
 PART : P A R T;
 PARTITION : P A R T I T I O N;
 PARTITIONED: P A R T I T I O N E D;
+PARTITIONING: P A R T I T I O N I N G;
```

Comment:
Does this new token need to be added to `dbs_special_name`?

Reply:
added at line 1598 of parser